modules/component-store/src/component-store.ts

```diff
@@ -61,6 +71,19 @@ export class ComponentStore<T extends object> implements OnDestroy {
   // Needs to be after destroy$ is declared because it's used in select.
   readonly state$: Observable<T> = this.select((s) => s);
 
+  // check/call store init hook
+  private readonly initStoreHook = this.effect(() =>
```

Comment:
It seems like a comment didn't come through.
Why is `effect` used here, in comparison to the other hook where a manual `subscribe` and `take(1)` is used?

Reply:
I wanted to use the effect to handle the `subscribe`, but that did create some inconsistency. I changed them both to manual subscribes inside the methods.

Reply:
Thanks Brandon, also a good catch to make those methods private 👍